sdk/src/manifest_store.rs

```diff
@@ -76,6 +82,30 @@ impl ManifestStore {
         self.manifests.get(label)
     }
 
+    // writes a resource identified uri to the given stream
+    pub fn get_resource(&self, uri: &str, stream: impl Write + Read + Seek + Send) -> Result<u64> {
```

Comment:
This would only work with a jumbf uri from the manifest store. It is meant for retrieving resources from the store. I could add some error checking for that. As it is I think you would just get a Not Found result.